src/app.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
The app.py is not used anymore, I deleted it in my previous PR, the code to work with are in `src/fbosbot/__init__.py`

Reply:
I saw the code in init and wondered why it was there.
init should be empty, to begin with. although it is not bad to have code in there.

I needed to separate the db code to avoid circular imports, that is why I had to create app.py and manage.py

Reply:
Yes the db code is in a separated file, I follow this Flask tutorial to improve the code structure : try to do it : https://flask.palletsprojects.com/en/1.1.x/tutorial/ and you will understand the change I made.

Reply:
@elinguiuriel Looks like you followed it perfectly. Not sure why I should use it if I am using a totally different format from what is in the tutorial. notice that I am using ORM's and flask script and flask manage. These two dependencies do the same thing your create app does.